q2_sapienns/plugin_setup.py

```diff
@@ -56,6 +56,8 @@ def _equal_number_of_columns(self, n_lines):
                     break
                 fields = line.strip().split('\t')
                 n_fields = len(fields)
+                if file_length == idx + 1 and line == '\n':
+                    break
```

Comment:
@brett-van-tussler, I don't see `file_length` defined here. Did this change work for you? 

I think a better way to handle this would be as follows:

```suggestion
                if n_fields == 0:
                    continue
```

That will skip blank lines where ever they occur.